### DISCUSSION TITLE
Unable to embed superset dashboard in an iframe from an angular application

### DISCUSSION THREAD

**Discussion by HariniPuppala** [apache/superset#18313] [Q&A / Help]
Hi,
I am trying to integrate superset dashboard into an angular based web application by embedding superset dashboard in an iframe.
I did the following setup:
Configured LDAP Authentication in superset.
Enabled a Custom Security Manager to authenticate based on a jwt-token. The jwt-token is decoded and the credentials are validated as per the configured LDAP settings in the Custom Security manager.
The jwt-token based authentication is working as expected and I am able to redirect to the dashboards without the login page.  But, the same is not working when passed in an iframe.

Getting the below error in iframe:
![MicrosoftTeams-image](https://user-images.githubusercontent.com/87519153/125927995-f3d28684-dcc0-4a18-8ca3-348881370dfe.png)

There is this warning message in superset log:
WARNING:root:Class 'werkzeug.local.LocalProxy' is not mapped

Can anyone please help on this. Thanks in advance!

**Comment by ghost** [comment]
I am also getting the same error after upgrading superset from 1.1.0. to  1.2.0
Does anyone know about this error.

**Comment by gokultrivandrum** [comment]
@HariniPuppala @srikark16  Any solution for the error, Also other than iFrame is there any client JS for integrating Dashboard ? @junlincc

**Comment by jhult** [comment]
Also related to #16718.
